minithesis.py

```diff
@@ -82,16 +82,15 @@
 S = TypeVar("S", covariant=True)
 U = TypeVar("U")  # Invariant
 
+InterestTest = Callable[[array], bool]  # Really array[int] -> bool
+
 
 class Database(Protocol):
-    def __setitem__(self, key: str, value: bytes) -> None:
-        ...
+    def __setitem__(self, key: str, value: bytes) -> None: ...
```

Comment:
What's going on here? I'm not a fan of it.

Reply:
Ah, these slipped through. `black` has been putting things like this, and I've been putting it back every single commit. Maybe some kind of line-length parameter thing?